docs/features.md

```diff
@@ -375,6 +375,27 @@ async def route_before_request(request: Request):
     request.headers["before"] = "route_before_request"
 ```
 
+In the before middleware, you can choose to directly return a `Response` object. When doing so, the route method and the after middlewares will not be executed.
+
+```python
+def is_user_logged(request: Request):
+    # Check the validity of a JWT cookie for example
+    ...
+
+@app.before_request("/")
+async def hello_before_request(request: Request):
```

Comment:
@AntoineRR , can we just mention here as a comment or something that the request will be aborted here?

Reply:
Sure, I just updated it :)